geostore/views/__init__.py

```diff
@@ -152,6 +152,24 @@ def partial_update(self, request, *args, **kwargs):
         else:
             return HttpResponseBadRequest(_('Features are missing in GeoJSON'))
 
+    @action(detail=True, methods=['get'])
+    def property_values(self, request, pk):
+        property_to_list = request.query_params.get('property')
+        if not property_to_list:
+            return HttpResponseBadRequest(content=_('Invalid "property" GET parameter'))
```

Comment:
It's valid but I suggest to keep drf Response class with 400 status

Reply:
And done.